tests/test_s3_utils.py

```diff
@@ -45,6 +55,12 @@
 else:
     ClientErrorResponseError = ClientErrorResponseTypeDef = dict
 
+SINGLE_CHUNK_ETAG_LENGTH = 34  # single chunk etag is "32_char", (+2) with quotes as part of etag
```

Comment:
What is "32_char"? If you mean "32 characters", it's probably easier to say so. You might also refer to a definition of ETag for both constants, to provide more info if needed.